flarefly/data_handler.py

```diff
@@ -29,6 +29,8 @@ def __init__(self, data=None, var_name='', limits=None, use_zfit=True, **kwargs)
             Limits of the x axis used in the fit
         use_zfit: bool
             If True, zfit package is used to fit the data
+        nbins: int
+            Number of bins chosen by user to bin data in case of unbinned data
         **kwargs: dict
             Additional optional arguments:
 
```

Comment:
```suggestion
        **kwargs: dict
            Additional optional arguments:

            - nbins: int
                Number of bins chosen by user to bin data in case of unbinned data

```